contracts/interfaces/0.8.x/IFilteredMinterV2.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "./IFilteredMinterV1.sol";
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title This interface extends the IFilteredMinterV1 interface in order to
+ * add support for manually setting project max invocations.
+ * @author Art Blocks Inc.
+ */
+interface IFilteredMinterV2 is IFilteredMinterV1 {
+    /**
+     * @notice Local max invocations for project `_projectId`, tied to core contract `_coreContractAddress`,
+     * updated to `_maxInvocations`.
+     */
+    event ManuallySetProjectMaxInvocations(
+        uint256 indexed _projectId,
+        address indexed _coreContractAddress,
```

Comment:
In our current minter suite, we heavily rely on each minter being tied to a single core contract address. It is a relationship that we will likely remove in some future iteration, but for now, I think we should continue to use that assumption, and this address parameter could be removed from this event.